resources/lang/no/app.php

```diff
@@ -71,5 +71,23 @@
     ],
     'trigger' => [
         'message' => 'Connecting to the shelter, please wait...(no)'
+    ],
+    'tac' => [
+        'default' => '		
```

Comment:
I think I would prefer if it was split into 
`'tac' => 
  'default' => [
    'title' => 'Brukervilkår',
    'intro_text1' => 'Ved installasjon ...',
    'intro_text2' => 'Dersom du samtykker ...',
    'subheading1 => 'Hva skjer når jeg ....'
   etc.
]` so it would be easier to construct the html dynamically. But if this is just a temporary placeholder until we let schools dynamically edit WYSIWYG T&Cs in eg. tinyMCE then I suppose it's fine.

Reply:
Yes. RTE later.